src/main/java/chess/Board.java

```diff
@@ -1,22 +1,107 @@
 package chess;
 
+import chess.pieces.Color;
 import chess.pieces.Pawn;
 
 import java.util.ArrayList;
+import java.util.List;
 
 public class Board {
+    private final List<Pawn> whitePawns = new ArrayList<>();
+    private final List<Pawn> blackPawns = new ArrayList<>();
 
-    ArrayList<Pawn> pawns = new ArrayList<>();
+    public static final int BOARDSIZE = 8;
+    private final String blank = "........";
 
-    public void add(Pawn pawn) {
-        pawns.add(pawn);
+    public void addWhitePawn(Pawn pawn) {
+        if (pawn.getColorName().equals(Color.WHITE.getColorName())) {
+            whitePawns.add(pawn);
+        } else {
+            System.out.println("알 수 없는 color의 pawn입니다.");
+            System.out.println("add에 실패하였습니다.");
+        }
```

Comment:
```suggestion
        if (pawn.getColorName().equals(Color.WHITE.getColorName())) {
            whitePawns.add(pawn);
            return;
        } 
        System.out.println("알 수 없는 color의 pawn입니다.");
        System.out.println("add에 실패하였습니다.");
```
early return이라는 방법입니다~